pybossa/api/task_run.py

```diff
@@ -221,7 +219,7 @@ def update_quiz(task_id, project_id, answer):
         return
 
     task = task_repo.get_task(task_id)
-    if task.gold_answers == answer:
+    if get_gold_answers(task) == answer:
```

Comment:
if we retrieve gold answers earlier, they can be used for both quiz and stats

Reply:
Fixed